### DISCUSSION TITLE
Transform ImageToTransducerOrigin

### DISCUSSION THREAD

**Discussion by stka85** [PlusToolkit/PlusLib#1162] [General]
Hi, I'm not sure how to estimate the transformation _From="Image" To="TransducerOriginPixel"_. Could you please explain? Thank you!

**Comment by Sunderlandkyl** [comment]
These page has information on fCal coordinate systems:
http://perk-software.cs.queensu.ca/plus/doc/nightly/user/CommonCoordinateSystems.html
http://perk-software.cs.queensu.ca/plus/doc/nightly/user/ApplicationfCalCoordinateSystemDefinitions.html

So the ImageToTransducerOriginPixel will be a translation that goes from the origin of the image (probably one of the corners of the image), to the position of the middle of the transducer in the first line of the image. Both coordinate systems are in Pixel coordinates, so no scaling needs to be applied.

**Reply by stka85** [reply]
Thank you for your response! I do not understand why the example has ty=5 in the translation matrix, when the position is in the first line of the image. http://perk-software.cs.queensu.ca/plus/doc/nightly/user/ApplicationfCal.html

**Reply by Sunderlandkyl** [reply]
Not sure without seeing the original ultrasound image, but I would guess that the B-mode image doesn't actually start until the 5th line of pixels.
